public/beacon/catcher.php

```diff
@@ -25,6 +25,9 @@
 
 
 use App\BasicRum\Beacon\Catcher\Storage\File;
+use App\BasicRum\ResourceTimingDecompressor_v_0_3_4;
+
+$decompressor = new ResourceTimingDecompressor_v_0_3_4();
```

Comment:
In catcher part we try to save a beacon as soon as possible and we try to avoid any pre-calculations. Best is if we handle such tasks in the import process. Let's sit together and work on this ;)

Reply:
Okay. Lets talk about it